README.md

```diff
@@ -141,3 +141,51 @@ This repository is available under the terms of the Apache 2.0 license.
 View the [LICENSE file](LICENSE) for more information.
 
 Copyright (c) 2017 BBC
+
+
+New Yaml
```

Comment:
If we're adding the Yaml for how to bring this thing into Symfony it probably needs describing a bit, especially what it's doing with caching etc.